code/src/main/java/frc/robot/Robot.java

```diff
@@ -32,19 +32,18 @@ public Robot() {
     // be written to the roborio's SD card/internal storage, which has limited write endurance
     DataLogManager.start();
     Epilogue.configure(config -> {
-      if (isSimulation()) {
+      if (isSimulation() || true) {
```

Comment:
this will need to be changed for comp possibly, depending on the network usage of logging but its fine for now